cumulusci/tasks/metadata_etl/picklists.py

```diff
@@ -0,0 +1,185 @@
+from typing import Dict
+
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.tasks.metadata_etl import MetadataSingleEntityTransformTask
+from cumulusci.utils.xml.metadata_tree import MetadataElement
+from cumulusci.core.utils import process_bool_arg, process_list_arg
+
+
+class AddPicklistEntries(MetadataSingleEntityTransformTask):
+    entity = "CustomObject"
+    task_options = {
+        "picklists": {
+            "description": "List of picklists to affect, in Object__c.Field__c form.",
+            "required": True,
+        },
+        "entries": {
+            "description": "Array of picklist values to insert. "
+            "Each value should contain the keys 'fullName', the API name of the entry, "
+            "and 'label', the user-facing label. Optionally, specify `default: True` on exactly one "
+            "entry to make that value the default. Any existing values will not be affected other than "
+            "setting the default (labels of existing entries are not changed).",
+            "required": True,
+        },
+        "record_types": {
+            "description": "List of Record Type developer names for which the new values "
+            "should be available. If any of the entries have `default: True`, they are also made "
+            "default for these Record Types. Any Record Types not present in the target org will be "
+            "ignored, and * is a wildcard. Default behavior is to do nothing."
+        },
+        "sorted": {
+            "description": "Set whether or not to sort the picklist values alphabetically. Default is no change."
+        },
+        **MetadataSingleEntityTransformTask.task_options,
+    }
+
+    def _init_options(self, kwargs):
+        self.task_config.options["api_names"] = "dummy"
```

Comment:
?

Reply:
Quirk of the Metadata ETL framework. To be refactored when I pull out the service tasks to build Enablement Scripts.